action.yml

```diff
@@ -96,6 +102,17 @@ runs:
 
       run: ${{ github.action_path }}/scripts/digest_inputs 
 
+    - name: Docker Login - docker.artifactory.hashicorp.engineering(RO)
```

Comment:
This is an end-of-week 🧠 question/thought,  will logging in to the Artifactory docker mirror here have any side-effects in the calling workflows?

Reply:
For steps in the same job, the login will persist, so yes it would cause the calling job to remain logged in. That by itself is probably not too much of a problem. However, in general having this action reliant on authentication is something we want to avoid I think (see above).